### PR TITLE
setting Intel bach as default compiler for Levante. Replacing env/lev…

### PR DESCRIPTION
**Pull request by dsidoren**
…ante.dkrz.de/shell.nvhpc->shell will allow to compile with NV & OpenACC support

### CODE REVIEW

File: env/levante.dkrz.de/shell
Comment (suvarchal) [inline env/levante.dkrz.de/shell:1]: does it make sense to add multio directories to it ?

**Review by suvarchal — CHANGES_REQUESTED** [review]
may be multio directory pointing to a user's directory is not elegant?

**Review by koldunovn — APPROVED** [review]

File: env/levante.dkrz.de/shell
Comment (suvarchal) [reply env/levante.dkrz.de/shell:1]: resolving for now as it is harmless

**Review by suvarchal — APPROVED** [review]
moving on for now

**Pull request merged**